ts/parts/Dynamics.ts

```diff
@@ -376,7 +376,9 @@ extend(Chart.prototype, /** @lends Highcharts.Chart.prototype */ {
         (chartOptions as any)[type].push(userOptions);
 
         if (isColorAxis) {
-            this.isDirtyLegend = true;
+            this.series.forEach(function (series: Highcharts.Series): void {
+                series.update({}, false);
```

Comment:
Is it necessary to call `series.update()` on every series in the chart in order to make it bind to the color axis? Can we do this in a cheaper way?

Reply:
Yes, it looks like that we can use only `bindAxes` method:

        if (isColorAxis) {
            this.isDirtyLegend = true;
            this.series.forEach(function (series: Highcharts.Series): void {
                series.bindAxes();
            });
        }